### PR TITLE
add support for logos

### PR DESCRIPTION
**Pull request by rodrigomelo9**
The theme now supports logos, stored at static/logos

### CODE REVIEW

File: layouts/_default/list.html [hunk 1]
```diff
@@ -2,28 +2,30 @@
...
           {{- partial "icons.html" . -}}
         </div>
-        <div class="level-right">
-        </div>
       </nav>
     </div>
+    {{ $dir := "static/logos/" }}
+    {{ $img := lower (print .File.BaseFileName ".png") }}
+    {{ if (where (readDir $dir) "Name" $img) }}
+      <div class="level-right">
```
Comment (umarcor) [inline layouts/_default/list.html]: Please, use `media-right` instead: http://bulma.io/documentation/layout/media-object/

File: layouts/_default/list.html [hunk 2]
```diff
@@ -2,28 +2,30 @@
...
         <div class="level-left">
           {{- partial "icons.html" . -}}
         </div>
-        <div class="level-right">
-        </div>
       </nav>
     </div>
+    {{ $dir := "static/logos/" }}
+    {{ $img := lower (print .File.BaseFileName ".png") }}
+    {{ if (where (readDir $dir) "Name" $img) }}
```
Comment (umarcor) [inline layouts/_default/list.html]: I think you want use `with` here: https://gohugo.io/functions/with/

File: layouts/_default/single.html [hunk 3]
```diff
@@ -1,14 +1,31 @@
...
-  <h1 class="title">
-    {{ .Title }}
-  </h1>
-  <h2 class="subtitle">
-    {{ .Description }}
-  </h2>
-  <h3 class="subsubtitle">
-    {{ with .Params.Author }}Maintained by: {{ delimit . ", " }}{{ end }}
-  </h3>
+  <div class="columns">
```
Comment (umarcor) [inline layouts/_default/single.html:3]: You are dividing the vertical space in two equally sized columns. That produces the following result:

image

I think we want the second column (L15) to be narrow: https://bulma.io/documentation/columns/sizes/#narrow-column

File: layouts/_default/list.html [hunk 4]
```diff
@@ -2,28 +2,30 @@
...
+      <div class="level-right">
+        <figure class="image is-96x96">
+          <img src="{{ .Site.BaseURL }}logos/{{ $img }}" alt="{{ .File.BaseFileName }}">
```
Comment (umarcor) [inline layouts/_default/list.html]: Maybe logos should be a link to the details page: `<a href="{{.Permalink }}">`

File: layouts/_default/list.html (hunk 2 above)
Comment (rodrigomelo9) [reply layouts/_default/list.html]: Sorry, I made a lot of attempts, but I not figured out how to use *with* in this case (remember, I am not an expert with Hugo)

**Comment by umarcor** [comment]
Thanks!

**Pull request merged**